### PR TITLE
ticket 6 complete

### PR DESCRIPTION
**Pull request by sinnedchan**

### CODE REVIEW

File: src/components/ArticleCard.jsx [hunk 1]
```diff
@@ -1,11 +1,39 @@
-export default function ArticleCard({ article }) {
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getArticleById } from "../api";
+
+export default function ArticleCard() {
+  const { article_id } = useParams();
+  const [currArticle, setCurrArticle] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
+  const [isValid, setIsValid] = useState(true);
```
Comment (HMelesi) [inline src/components/ArticleCard.jsx:9]: Doesn't look like this is really ever false yet? If this is just forward planning for error handling on ticket 12 then that's fine, but it's not currently checking validity.

File: src/components/ArticleCard.jsx [hunk 2]
```diff
@@ -1,11 +1,39 @@
...
+  }
   return (
-    <li key={article["article_id"]}>
-      <h3>{article.title}</h3>
+    <div key={currArticle["article_id"]}>
+      <h3>{currArticle.title}</h3>
       <p>
-        {article["author_id"]} {article.topic} {article["comment_count"]}
-        {article.votes}
+        {currArticle.author} {currArticle["created_at"]}
```
Comment (HMelesi) [inline src/components/ArticleCard.jsx]: could use currArticle.created_at instead

File: src/components/ArticleCard.jsx [hunk 3]
```diff
@@ -1,11 +1,39 @@
...
       </p>
-    </li>
+      <p>{currArticle.topic} </p>
+      <p>{currArticle.body} </p>
+      <p>comments: {currArticle["comment_count"]} </p>
```
Comment (HMelesi) [inline src/components/ArticleCard.jsx]: same for this! Dot notation would work

**Review by HMelesi — APPROVED** [review]
Looks good Dennis! A couple of points about dot notation and states not being used but this is approved.

File: src/components/ArticleCard.jsx (hunk 1 above)
Comment (sinnedchan) [reply src/components/ArticleCard.jsx:9]: yep just setting out any potential error handling i might encounter later on

**Pull request merged**